src/components/Footer.jsx

```diff
@@ -46,32 +46,40 @@ const Footer = (props) => {
             </div>
             <div className="media-links">
               <div className="first-row">
-                <img
-                  src={
-                    'https://d2vgampz89jm7o.cloudfront.net/_Sponsors+Light+Boxed+2.svg'
-                  }
-                  alt="hacktoberfest-sponsors-logo"
-                />
-                <img
-                  src={
-                    'https://d2vgampz89jm7o.cloudfront.net/mattermore_logo.jpeg'
-                  }
-                  alt="mattermore-logo"
-                  id="square"
-                />
+                <a href="https://organize.mlh.io/participants/events/4616-project-xcov19">
```

Comment:
change href to https://hacktoberfest.cloud.mattermost.com/main/channels/mlh-4616